### PR TITLE
Instant stuns suck my nuts

### PR DESCRIPTION
**Pull request by Toxici11i**
Bugfixing stuns for other items now..

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Is this ready?

**Comment by Toxici11i** [comment]
yes

**Comment by qwertyquerty** [comment]
You sure you don't wanna nerf flashes while you're at it?

**Comment by Toxici11i** [comment]
on it bwerty

**Comment by qwertyquerty** [comment]
In my opinion they should make the white screen of you blind for a lot longer.

**Comment by qwertyquerty** [comment]
I don't think this is the answer for flashes. In my opinion we need to completely rethink what they do

**Comment by Toxici11i** [comment]
flashes are a cancer mechanic and need to be reworked, but this should temporarily make them not the solution to every situation TM

**Comment by qwertyquerty** [comment]
But why in the world would being blinded make you lose stamina

**Comment by Toxici11i** [comment]
Disorientation makes you drop items, slows you already (this also stops the cancer wall slam effect) etc, ill probably adjust this in a later PR

**Comment by qwertyquerty** [comment]
Why not in this one

**Comment by Toxici11i** [comment]
done

File: code/modules/assembly/flash.dm [hunk 1]
```diff
@@ -126,7 +126,9 @@
 				to_chat(M, "<span class='userdanger'>[user] blinds you with the flash!</span>")
 			else
 				to_chat(M, "<span class='userdanger'>You are blinded by [src]!</span>")
-			M.Paralyze(rand(80,120))
+			M.adjustStaminaLoss(rand(20,50))
+			if(prob(40))
```
Comment (qwertyquerty) [inline code/modules/assembly/flash.dm]: Great. Even more rng

Comment (Toxici11i) [reply code/modules/assembly/flash.dm]: prob(80)
still less RNG on the stuntime on the original flashcode

**Comment by qwertyquerty** [comment]
I still don't like this solution, as it's still stamina and rng based

**Comment by Toxici11i** [comment]
what would you suggest then? Making people drop items 100% of the time would be plain broken (see - flashers) and stamina can just be removed.

File: code/modules/assembly/flash.dm [hunk 2]
```diff
@@ -126,7 +126,7 @@
 				to_chat(M, "<span class='userdanger'>[user] blinds you with the flash!</span>")
 			else
 				to_chat(M, "<span class='userdanger'>You are blinded by [src]!</span>")
-			M.Paralyze(rand(80,120))
+				M.Paralyze(rand(80,120))
```
Comment (qwertyquerty) [inline code/modules/assembly/flash.dm]: You flubbed up the indentation also there's still RNG

File: code/modules/assembly/flash.dm [hunk 3]
```diff
@@ -126,7 +126,7 @@
 				to_chat(M, "<span class='userdanger'>[user] blinds you with the flash!</span>")
 			else
 				to_chat(M, "<span class='userdanger'>You are blinded by [src]!</span>")
-			M.Paralyze(rand(80,120))
+				M.Paralyze(70)
```
Comment (qwertyquerty) [inline code/modules/assembly/flash.dm]: You still messed up the indentation here

Comment (qwertyquerty) [reply code/modules/assembly/flash.dm]: You messed up the indentation even more

**Pull request merged**